### ISSUE TITLE
[Feature Request] Add type declaration file

### ISSUE THREAD

**Issue by mizlan** [jorenvanhee/tailwindcss-debug-screens#25]
<img width="1451" alt="Screenshot 2023-12-15 at 2 55 01 PM" src="https://github.com/jorenvanhee/tailwindcss-debug-screens/assets/44309097/a6a721ff-ad9e-4e1b-ae22-0c68b2ba9fc1">

Getting the warning, was wondering if you'd consider adding the type declaration file.

**Comment by jorenvanhee** [comment]
Good idea :) I guess it would be more or less the same like the type declaration file in the tailwindcss/typography plugin? Would be more than happy to merge a PR.

**Comment by mizlan** [comment]
I'll send a PR soon!

**Comment by mizlan** [comment]
As I was browsing some documentation, I was curious why you specify configuration to be defined in `theme` instead of in an options argument like Typography:

https://tailwindcss.com/docs/typography-plugin#changing-the-default-class-name

Seems like the latter makes more sense, since there is locality between the plugin and the configuration

**Comment by jorenvanhee** [comment]
I built this plugin back in the Tailwind CSS 1.x days and that happened to be the [recommended way](https://v1.tailwindcss.com/docs/plugins#exposing-options). I see that they now recommend [another way](https://tailwindcss.com/docs/plugins#exposing-options) indeed.

I guess it would be easier to add the types once the new recommended options are implemented?

**Comment by mizlan** [comment]
Yeah, probably? I have a file that is sufficient to stop the warning in my original screenshot but I can add another call with an options object and it doesn't error. Don't know if that can be prevented.
